src/app/src/mainwindow.cpp

```diff
@@ -216,7 +217,7 @@ MainWindow::MainWindow(std::shared_ptr<Config> params, QMainWindow *parent) : QM
         m_acquisitionProgress->setCancelButton((msg.contains("Acquiring images") && m_config->triggerMode == EXT_TRIG_TRIG_FIRST) ? new QPushButton("&Trigger", this) : nullptr);
     });
 
-    connect(this, &MainWindow::sig_progress_update, this, [this](size_t n) {
```

Comment:
probably need more than unit16_t for progress if recordings can go longer than 5 min